### ISSUE TITLE
Remove StringBuilder in Logback TimestampConverter

### ISSUE THREAD

**Issue by KarstenSchnitter** [SAP/cf-java-logging-support#90]
[TimestampConverter](https://github.com/SAP/cf-java-logging-support/blob/master/cf-java-logging-support-logback/src/main/java/com/sap/hcp/cf/logback/converter/TimestampConverter.java) in cf-java-logging-support-logback contains the following code:

```java
    @Override
    public String convert(ILoggingEvent event) {
        StringBuilder appendTo = new StringBuilder();
		Instant now = Instant.now();
		long timestamp = now.getEpochSecond() * 1_000_000_000L + now.getNano();
		appendTo.append(timestamp);
        return appendTo.toString();
    }
```

Creating the `StringBuilder` is a time consuming implementation for creating a string representation of the `long` timestamp. This should be changed to a more efficient implementation.

**Comment by wolches** [comment]
Is this issue actual yet?

**Comment by KarstenSchnitter** [comment]
@wolches Yes, this is a current open issue. I came across it during a small performance analysis of the library. Since the code is executed for every emitted message, improvements have a larger impact.

**Comment by KarstenSchnitter** [comment]
Resolved by #97
